code/modules/lighting/lighting_system.dm

```diff
@@ -338,10 +338,7 @@
 		if(newalpha >= LIGHTING_DARKEST_VISIBLE_ALPHA)
 			newalpha = 255
 		if(lighting_object.alpha != newalpha)
-			if(instantly)
```

Comment:
most likely. It doesn't always treat characters like `_` or `/` sanely, so searching procs on it can be a pain.